docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -11,12 +11,110 @@ menu:
 aliases:
 type: page
 isTocNested: true
-showAsideToc: false
+showAsideToc: true
 ---
 
-You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+Yugabyte Platform allows you to use its UI or API to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between two data centers.
 
-If you are using Yugabyte Platform to manage universes, you need to call the following REST API endpoint on your Yugabyte Platform instance for the producer universe and the consumer universe involved in the asynchronous replication between two data sources:
+Within the concept of replication, universes are divided into the following categories:
+
+- A source universe contains the original data that is subject to replication.
+
+  Note that in the current release, replicating a source universe that has already been populated with data can be done only by contacting Yugabyte Support.
+
+- A target universe is the recepient of the replicated data. One source universe can replicate to one or more target universes.
+
+For additional information on asynchronous replication in YugabyteDB, see [Asynchronous Replication: Overview and Architecture](https://docs.yugabyte.com/latest/architecture/docdb-replication/async-replication/).
+
+## Use the Yugabyte Platform UI
+
+You can use the Yugabyte Platform UI to set up and configure asynchronous replication. In addition, you can perform monitoring by accessing the information about the replication lag and enabling alerts on excessive lag.
+
+### How to set up replication
+
+You can set up asynchronous replication for a universe which does not contain data as follows:
+
+1. Open the Yugabyte Platform UI and navigate to **Universes**.
+
+2. Select the universe you want to replicate and navigate to **Replication**.
+
+3. Click **Configure Replication** to open the dialog shown in the following illustration:<br><br>
+
+   ![Configure Replication](/images/yp/asynch-replication-2.png)<br><br>
+
+4. Provide the name for your replication.
+
+5. Select the target universe.
+
+6. Click **Next: Select Tables**.
+
+7. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Create Replication**, as per the following illustration:<br><br>
+
+   ![Create Replication](/images/yp/asynch-replication-3.png)
+
+### How to configure replication
+
```

Comment:
Any constraints on configuring? can you configure a universe that has data and is running, or is it like setup and you need it to be empty? Setup and Configuration seem identical, can they just be combined in 1 section?

Reply:
Tables need to be empty if the replication is done via the UI. 
The starting point of "Configure" is different from "Set up". I'll keep these sections separate.